core/src/main/scala/com/evolutiongaming/catshelper/LogOf.scala

```diff
@@ -21,9 +21,9 @@ object LogOf {
 
   def summon[F[_]](implicit F: LogOf[F]): LogOf[F] = F
 
-  def logger[F[_]: LogOf, C: ClassTag]: F[Log[F]] = apply[F].apply(implicitly[ClassTag[C]].runtimeClass)
+  def log[F[_]: LogOf, C: ClassTag]: F[Log[F]] = apply[F].apply(implicitly[ClassTag[C]].runtimeClass)
```

Comment:
regarding this - I propose to add extension method to `Log` class, and reuse that method, so `ClassTag` functionality is available in both cases

Reply:
done